www/docs/en/dev/guide/platforms/android/webview.md

```diff
@@ -38,97 +38,56 @@ legacy `CordovaActivity` component that pre-dates the 1.9 release.
    [cordova.apache.org](http://cordova.apache.org) and unzip its
    Android package.
 
-1. Navigate to the Android package's `/framework` directory and run
-   `ant jar`. It creates the Cordova `.jar` file, formed as
-   `/framework/cordova-x.x.x.jar`.
-
-1. Copy the `.jar` file into the Android project's `/libs` directory.
-
-1. Add the following to the application's `/res/xml/main.xml` file,
-   with the `layout_height`, `layout_width` and `id` modified to suit
-   the application:
-
-        <org.apache.cordova.CordovaWebView
-            android:id="@+id/tutorialView"
-            android:layout_width="match_parent"
-            android:layout_height="match_parent" />
-
-1. Modify the activity so that it implements the `CordovaInterface`.
-   It should implement the included methods.  You may wish to copy
-   them from `/framework/src/org/apache/cordova/CordovaActivity.java`,
-   or else implement them on your own.  The following code fragment
-   shows a basic application that relies on the interface. Note how
-   the referenced view id matches the `id` attribute specified in the
-   XML fragment shown above:
-
-        public class CordovaViewTestActivity extends Activity implements CordovaInterface {
-            CordovaWebView cwv;
-            /* Called when the activity is first created. */
-            @Override
-            public void onCreate(Bundle savedInstanceState) {
-                super.onCreate(savedInstanceState);
-                setContentView(R.layout.main);
-                cwv = (CordovaWebView) findViewById(R.id.tutorialView);
-                Config.init(this);
-                cwv.loadUrl(Config.getStartUrl());
-            }
-
-1. If the application needs to use the camera, implement the
```

Comment:
Does this still apply?

Reply:
Config.getStartUrl() is marked as deprecated and has changed. I didn't touch the camera but assume it has changed as well.